services/state/roster/ledger_id.proto

```diff
@@ -74,14 +85,15 @@ message RosterSignatures {
   bytes roster_hash = 1;
 
   /**
-   * A list of node signatures on the same message where all node ids in the NodeSignature objects are from the
-   * roster that the roster_hash represents.
+   * A list of node signatures on the same message where all node ids in the
+   * NodeSignature objects are from the roster that the roster_hash represents.
    */
   repeated NodeSignature node_signatures = 2;
 }
 
 /**
- * A pair of a _RSA_ signature and the node id of the node that created the signature.
+ * A pair of a _RSA_ signature and the node id of the node that created the
```

Comment:
This is still missing any meaningful content for requirements, block stream effects, etc... A single sentence half descriptor is no longer "good" for messages in proto files, we should fully specify the `NodeSignature` API here (this includes the fields below, and all other messages/fields in this PR).